Source/FakeItEasy.Specs/LazySpecs.cs

```diff
@@ -23,19 +25,37 @@ public interface IFoo
             ILazyFactory fake,
             Lazy<IFoo> lazy)
         {
-            "establish"
+            "Given a fake"
                 .x(() => fake = A.Fake<ILazyFactory>());
 
-            "when calling a method that returns a lazy"
+            "When calling an unconfigured method that returns a lazy of a fakeable type"

```

Comment:
Technically it's "dummyability" that's the determining factor, not fakeability. That's a bit of a quibble here, since every fakeable type is dummyable. But meet me below.